deny.toml

```
@@ -19,6 +19,7 @@ allow = [
     "ISC",
     "MIT",
     "OpenSSL",
+	"Unicode-DFS-2016"
```

nit: We should probably keep the tab spacing consistent
```suggestion
   "Unicode-DFS-2016"
```